lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -21,16 +21,40 @@ class RenderForm extends React.Component {
       currentRequest: null,
       errorList: [],
       success: false,
+      show_details: true,
     };
     this.submitForm = this.submitForm.bind(this);
+    this.reDirectUser = this.reDirectUser.bind(this);
     this.setErrorState = this.setErrorState.bind(this);
+    this.setDisplay = this.setDisplay.bind(this);
   }
 
   setErrorState(errors) {
     this.setState({
       errorList: errors,
     });
   }
+  setDisplay(event){
+   event.target.value === "Course Content" ? this.setState({
+            show_details: false,
+          }) : this.setState({
+            show_details: true,
+          })
+
+  }
+
+  setDisplay(event) {
+    event.target.value === "Course Content" ? this.setState({
+            show_details: false,
+          }) : this.setState({
+            show_details: true,
+          })
+
+  }
+
+  reDirectUser(){
+
```

Comment:
Take a look at `react-router-dom` https://reactjs.org/docs/code-splitting.html#route-based-code-splitting

Reply:
Also, I'm not sure what you're trying to do with routes, is it to redirect the user to the discussion for that course? Isn't just a link?

Reply:
yep, I'm trying to redirect a user to discussions. When the function "reDirectUser" is called, it will somehow redirect people to discussions